src/simpleNbox.cpp

```diff
@@ -145,14 +155,20 @@ void SimpleNbox::setData( const std::string &varName,
         else if( varNameParsed == D_VEGC ) {
             H_ASSERT( data.date == Core::undefinedIndex(), "date not allowed" );
             veg_c[ biome ] = data.getUnitval( U_PGC );
+            // HACK: Set value at t=0
+            veg_c_tv.set(0, veg_c);
```

Comment:
So, is this about making sure that setting `veg_c` works if it happens at some time after startup?  And before we were assuming that this would only be set from the ini file (not by message)?

Reply:
Without this line, `veg_c` is overwritten during reset by a line like `veg_c = veg_c_tv.get(t)`, which eliminates the newly-created biome.

Reply:
Thanks for clarifying.